LocalMultiplayerAgent/DeploymentTool/DeploymentSettings.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.Gaming.LocalMultiplayerAgent.DeploymentTool
+{
+    using System.Collections.Generic;
+    using PlayFab.MultiplayerModels;
+
+    public class DeploymentSettings
+    {
+        public string BuildName { get; set; }
+
+        public string VmSize { get; set; }
+
+        public int MultiplayerServerCountPerVm { get; set; }
+
+        public string OSPlatform { get; set; }
+
+        public List<string> AssetFileNames { get; set; }
```

Comment:
A neat trick I like with Lists in Json is to declare them like this:

public List<string> AssetFileNames {get; } => new List<string>()

The reason is that that way this will never be null. Json deserializers know how to insert into a list. 

now, if this is getting passed in as a new List() by someone else, disregard - this is fine. 

(applies everywhere you use lists in json)